app/views/transactions/_fields.html.erb

```diff
@@ -9,7 +9,7 @@
 
   <div class="form-group">
     <label for="edition_start_button_text">Start button text:</label><br/>
-    <% ["Start now", "Sign in"].each do |option| %>
+    <% ["Start now", "Sign in", "Dechrau nawr", "Mewngofnodi"].each do |option| %>
```

Comment:
Might be better as:
* Start now
* Dechrau nawr
* Sign in
* Mewngofnodi

The UI should probably still indicate somehow that it's the welsh translation.